views/forms/ControlledMultiSelect/index.jsx

```diff
@@ -1,8 +1,30 @@
 import React from 'react'
 import { Controller } from 'react-hook-form'
-import { Autocomplete, InputAdornment, TextField } from '@mui/material'
+import { Autocomplete, Chip, TextField } from '@mui/material'
 
 const ControlledMultiSelect = (props) => {
+  const onChange = (_, value, reason, field) => {
+    switch (reason) {
+      case 'clear':
+        value = field.value.filter(({ isFixed }) => isFixed)
+
+        return field.onChange(value)
+      case 'removeOption':
+        const optionToRemove = field.value.filter(
+          (prevOption) =>
+            value.findIndex(
+              (currentOption) => currentOption.value === prevOption.value
+            ) === -1
+        )
```

Comment:
I think you can clean this up a little by using `find` instead of `findIndex`

```suggestion
        const optionToRemove = field.value.filter((prevOption) =>
            value.find(currentOption => currentOption.value === prevOption.value)
        )
```

Reply:
They look pretty similar imo
[47a7352](https://github.com/AMP-SCZ/dpdash/pull/487/commits/47a7352aa90256c68d3515ee70b8a513b3052989)